utils/land-patterns.stanza

```diff
@@ -18,6 +20,58 @@ defn ensure-even-positive! (value: Int, name: String):
   if value < 2 or value % 2 == 1:
     fatal("%_ only supports a positive even number of pins." % [name])
 
+; ====== Silk screen helpers ===================================================
+; Create a polarity marker relative to a pad.
+; 
+; pad-1: the pad to mark for the polarity 
+; part-dims?: optional. If provided, the generator will use these to determine where
```

Comment:
This utility seems to assume that the current landing pattern is centered at the origin. Is that true? 

Can we remove this assumption? If not, this assumption makes this utility less applicable as a general user-facing utility.